vllm/entrypoints/openai/cli_args.py

```diff
@@ -289,6 +289,19 @@ def validate_parsed_serve_args(args: argparse.Namespace):
         raise TypeError("Error: --enable-reasoning requires "
                         "--reasoning-parser")
 
+    # Ensure that --max-num-batched-tokens, --max-num-seqs, --max-model-len
```

Comment:
Could you add some description about why we want to make sure these arguments are passed?